pkg/minishift/openshift/openshift.go

```diff
@@ -298,3 +301,22 @@ func getOpenshiftVersion() string {
 	}
 	return openshiftVersion
 }
+
+func GetOpenShiftVersionDuringStart() (string, error) {
+	tag := viper.GetString(configCmd.OpenshiftVersion.Name)
+	if tag == "vlatest" {
```

Comment:
:-s, are really using `vlatest` ?

Reply:
if this is the result of the prefix, please abstract this and combine using `package.VersionPrefix + "latest"`... or document wit comment